configurator/project/my-element.ts

```diff
@@ -0,0 +1,15 @@
+import {LitElement, html, property, customElement} from 'lit-element';
+
+@customElement('my-element')
+export class MyElement extends LitElement {
+  @property({type: Number})
+  myNumber?: number;
+
+  render() {
+    return html`
+      <div>Here is ${this.myNumber}</div>
+      <div>And here is my second element:</div>
+      <my-second-element></my-second-element>
```

Comment:
I'd remove this if it isn't included in the project

Reply:
Done.